### ISSUE TITLE
 OpenSSL is not found on macOS

### ISSUE THREAD

**Issue by erikbor** [riboseinc/retrace#319]
@jjr840430 

I just noticed that OpenSSL is not found on macOS during configure:
```sh
$ ./configure
<snip>
checking for OpenSSL... --with-openssl not specified
checking openssl/ssl.h usability... no
checking openssl/ssl.h presence... no
checking for openssl/ssl.h... no
configure: OpenSSL not found, OpenSSL support disabled
checking whether O_TMPFILE is declared... no
```

Can you fix this similar to how you did with RVC please?
And can you verify that the OpenSSL test/example still works.

**Comment by jjr840430** [comment]
@erikbor will do, Thanks.

**Comment by jjr840430** [comment]
@erikbor Fixed by #321
